abel/onion_dasch.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+import abel
+from scipy.linalg import inv
+from scipy import dot
+
+#################################################################################
+#
+#  Dasch onion-peeling deconvolution
+#    as described in Applied Optics 31, 1146 (1992), page 1148 sect. D.
+#    see PR #155
+#
+# 2016-03-25 Dan Hickstein - one line Abel transform
+# 2016-03-24 Steve Gibson - Python code framework
+# 2015-12-29 Dhrubajyoti Das - three_point code and highlighting the Dasch paper
+#                              see issue #61
```

Comment:
Maybe include the url for the issue. This code might work it's way into something completely different in the future, and it would be good to just have the link.